pandas/tests/api/test_api.py

```diff
@@ -63,7 +63,7 @@ class TestPDApi(Base):
     # top-level functions
     funcs = ['bdate_range', 'concat', 'crosstab', 'cut',
              'date_range', 'interval_range', 'eval',
-             'factorize', 'get_dummies', 'cdate_range',
+             'factorize', 'get_dummies',
```

Comment:
move to the deprecated functions section, and add a test (in same module), model after the top-level deprecations.